event-svc/benches/sqlite_store.rs

```diff
@@ -77,7 +77,7 @@ async fn model_setup(tpe: ModelType, cnt: usize) -> ModelSetup {
         rand::thread_rng().fill_bytes(&mut data);
 
         let (order_key, cid, event) = generate_init_event(&model, &data, signer.clone()).await;
-        events.push(EventInsertable::new(order_key, cid, event, true).unwrap());
+        events.push(EventInsertable::new(order_key, cid, Arc::new(event), true).unwrap());
```

Comment:
yeah, this comes into play in the next PR (#503). It's because Event (and therefore EventInsertable) aren't clone, and when we get a batch of events, we need to find the init event in the set in memory, or in the database. putting this in an Arc makes it easy/cheap to build the map of init events as `{cid: Arc<event>}` to find the controller/header information we need to validate the signature. It also makes it possible/easy to build a LRU cache of recently seen events that can be shared across different services.

Alternatively, we could probably have something that just takes the EventInsertable into the map and you can get it out via a reference, which is all that `store::insert_many` requires, but that might require a bit more thought as it would have to be inside a service wide `Arc<Mutex<>>` (or something with message passing) rather than letting each thread clone the pointers they need.